### PR TITLE
PayNow improvements

### PR DESCRIPTION
**Pull request by ribeiroguilherme**
## Summary
- Added `Timeline` component which display an instructions timeline
- PayNow component now renders the `Timeline` at the top in case it is mobile view. The breakpoint for mobile view is width < 1024px 
- Adjusted QR code component CSS according to our design (margins, paddings, font-sizes)
- Added PayNow, PayMe and Timeline storybook stories
- Added translations

## Tested scenarios
- Added unit tests
- Tested PayNow with desktop and mobile view

This PR will be added to the `v6-qr-code-improvements` branch which will contain other changes for other QR code payment methods as well, so we release them all together

### CODE REVIEW

File: packages/lib/src/components/PayNow/PayNow.test.ts [hunk 1]
```diff
@@ -21,4 +28,53 @@ describe('PayNow', () => {
...
+            loadingContext: 'checkoutshopper.com/',
+            modules: { resources, analytics: global.analytics, srPanel },
+            i18n: global.i18n,
+            paymentData: 'Ab02b4c0!BQABAgBH1f8hqfFxOvbfK..',
+            qrCodeImage: '',
+            paymentMethodType: 'paynow',
+            qrCodeData: '00020126580009SG...'
+        });
+
+        render(paynow.mount('body'));
```
Comment (longyulongyu) [inline packages/lib/src/components/PayNow/PayNow.test.ts:64]: Wouldn't `render(paynow.render())` not work?

File: packages/lib/src/components/internal/Timeline/Timeline.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
+@import 'styles/variable-generator';
+
+.adyen-checkout-timeline {
+  scrollbar-color: #c0c5cc transparent;
```
Comment (longyulongyu) [inline packages/lib/src/components/internal/Timeline/Timeline.scss:4]: Don't we have a semantic bento token for it?

File: packages/lib/src/components/internal/Timeline/Timeline.scss [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+.adyen-checkout-timeline__items {
+  all: unset;
+}
+
+.adyen-checkout-timeline-item {
+  scrollbar-color: var(--b-color-background-quaternary) transparent;
```
Comment (longyulongyu) [inline packages/lib/src/components/internal/Timeline/Timeline.scss:12]: Maybe use `token(color-background-quaternary)`?

File: packages/lib/src/components/internal/Timeline/Timeline.scss [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  max-width: token(spacer-070);
+  margin-top: token(spacer-010);
+}
+
+.adyen-checkout-timeline-item__separator {
+  flex-grow: 1;
+  position: relative;
+
+  &::before {
+    background-color: #c9cdd3;
```
Comment (longyulongyu) [inline packages/lib/src/components/internal/Timeline/Timeline.scss:47]: Same here to use a token?

**Review by longyulongyu — APPROVED** [review]

File: packages/lib/src/components/PayNow/PayNow.test.ts (hunk 1 above)
Comment (ribeiroguilherme) [reply packages/lib/src/components/PayNow/PayNow.test.ts:64]: It doesn't, there are errors about undefined props of SRPanel. I guess I need to properly mock the object for this test-case. I will try to fix it in the other PR

**Pull request merged**